Yosemite/YosemiteTests/Mocks/Networking/Remote/MockWooShippingRemote.swift

```diff
@@ -5,39 +5,49 @@ import XCTest
 ///
 final class MockWooShippingRemote {
 
-    private struct CreatePackageResultKey: Hashable {
+    private struct ResultKey: Hashable {
```

Comment:
Thanks for catching this! I just noticed I'd missed updating this before, too. 👍